tests/lean/run/bv_decide_rewriter.lean

```diff
@@ -292,6 +308,8 @@ example {x y z : BitVec 64} : ~~~(x &&& (y * z)) = (~~~x ||| ~~~(z * y)) := by
 
 example {x : BitVec 16} : (x = BitVec.allOnes 16) → (BitVec.uaddOverflow x x) := by bv_decide
 
+example {x : BitVec 16} : (x = BitVec.allOnes 16) → (BitVec.umulOverflow x x) := by bv_decide
+
```

Comment:
Not sure if we can have some more examples, with and without overflow. Maybe also checking is `smul` works.